flake.nix

```diff
@@ -50,15 +50,20 @@
             });
 
           # Overlay to specify build should use the specific bcc we want
-          bccVersion = "0.30.0";
+          #
+          # Waiting on a patch release to contain 68710c4f ("syms: Initialize
```

Comment:
Mind making an issue to switch us back - just so we don't forget?

Reply:
bcc just cut v0.33.0. I'll just bump to that.